src/main/app/controller/courts/LocalAuthoritiesController.ts

```diff
@@ -0,0 +1,193 @@
+import autobind from 'autobind-decorator';
+import {AuthedRequest} from '../../../types/AuthedRequest';
+import {Response} from 'express';
+import {CourtType} from '../../../types/CourtType';
+import {
+  LocalAuthoritiesAreaOfLaw,
+  LocalAuthoritiesPageData,
+  LocalAuthority,
+  LocalAuthorityItem
+} from '../../../types/LocalAuthority';
+import {AreaOfLaw} from '../../../types/AreaOfLaw';
+
+import {SelectItem} from '../../../types/CourtPageData';
+import {CSRF} from '../../../modules/csrf';
+
+export enum familyAreaOfLaw {
```

Comment:
I think generally if an enum is public, it should be in its own file. If it's private then it's cool. Would you mind either making it private or putting it in an enum folder of sorts? Thanks. Happy to discuss

Reply:
I have moved enum to its own folder under utils